### DISCUSSION TITLE
custom css

### DISCUSSION THREAD

**Discussion by gwmatthews** [damionx7/Zettel-Notes-Documentation#162] [Feature Requests]
It would be nice to be able to add a custom css file that overrides the default css -- ideally specified in the yaml header. This could be useful for creating simple pdf slideshows, for example, by setting the page size for printing to one suitable for slides.

**Comment by damionx7** [comment]
Hey! Currently zettel notes uses markwon library which is text view based and doesn't use webview. So, currently, no css is used for viewer mode. Maybe later, when we switch to webview based implementation, I will add this feature.

Edit: Unless you are talking about the print css ?

**Reply by gwmatthews** [reply]
Yes, I was talking about the print css. I know in Obsidian it is possible to add custom css classes in the yaml header and this appends those class names to the topmost <div> tag. I am not sure if that would do the trick, so the option to switch to a different custom css file instead of the default print css would be nice too!
